source/educators/how-tos/add_delete_course_tags

```diff
@@ -0,0 +1,46 @@
+Add and delete tags on courses
+##############################
+
+Adding Tags from the Course Outline Page
+****************************************
+
+#. On the Course Outline page, click on "Manage Tags" in the course header section. The tag drawer will open.
+
+   Insert image 1
+   Insert image 2
+
+#. Click “Edit Tags”.
+
+#. Click in the “Add a tag” field.
+   Insert image 3
+
+#. If you know the tag you would like to add, start typing the tag in the “Add a tag” field and matches will display.
+   Insert image 5
+
+#. If you don’t know the tag you would like to add, scroll through the list of tags until you find the tag you would like to add. 
+   **Note:** Tags may be nested as subtags, and that clicking on the arrows will open any nexted tags.
+   Insert image 6
+
+#. Once you have decided which tag you would like to add, click on the checkbox next to the tag. Add as many tags as you would like. 
+   **Note:** Choosing a nested tag will automatically add its parent tag(s) as well. 
+   Insert image 7
+
+#. When you are finished, click “Add tags”. You will see the newly added tags in the tag drawer view.
+   Insert image 8
+
+#. Click save and close out of the tag drawer. You will see the tag count updated in the Manage Course Tags field.
+   Insert image 9
+
+Deleting Tags from the Course Outline Page
+******************************************
+
+#. Click on “Manage Tags” in the course header section. The tag drawer will open.The tag drawer will open.
+   Insert image 10
+
+#. Click “edit tags”.
+
+#. Click the “x” button next to the tag you wish to delete.
+   **Note:** If you delete a tag that is nested underneath another tag, all tags in the hierarchy will delete.
```

Comment:
Might want to use the "Note" directive: https://docs.openedx.org/en/latest/documentors/references/quick_reference_rst.html#directives

Note (😄 ) that you need to be a bit careful about spacing when nesting a directive under a bullet.